pkg/metrics/metricsource.go

```diff
@@ -60,6 +60,149 @@ const (
+	MetricSourceCalico
+	MetricSourceCassandraNodetool
+	MetricSourceCeph
+	MetricSourceCertManager
+	MetricSourceCilium
+	MetricSourceCitrixHypervisor
+	MetricSourceClickhouse
+	MetricSourceCloudFoundryAPI
+	MetricSourceCockroachdb
+	MetricSourceConsul
+	MetricSourceCoredns
+	MetricSourceCouch
+	MetricSourceCouchbase
+	MetricSourceCrio
+	MetricSourceDirectory
+	MetricSourceDNSCheck
+	MetricSourceDotnetclr
+	MetricSourceDruid
+	MetricSourceEcsFargate
+	MetricSourceEksFargate
+	MetricSourceElastic
+	MetricSourceEnvoy
+	MetricSourceEtcd
+	MetricSourceExchangeServer
+	MetricSourceExternalDNS
+	MetricSourceFluentd
+	MetricSourceFoundationdb
+	MetricSourceGearmand
+	MetricSourceGitlab
+	MetricSourceGitlabRunner
+	MetricSourceGlusterfs
+	MetricSourceGoExpvar
+	MetricSourceGunicorn
+	MetricSourceHaproxy
+	MetricSourceHarbor
+	MetricSourceHdfsDatanode
+	MetricSourceHdfsNamenode
+	MetricSourceHTTPCheck
+	MetricSourceHyperv
+	MetricSourceIbmAce
+	MetricSourceIbmDb2
+	MetricSourceIbmI
+	MetricSourceIbmMq
+	MetricSourceIbmWas
+	MetricSourceIis
+	MetricSourceImpala
+	MetricSourceIstio
+	MetricSourceKafkaConsumer
+	MetricSourceKong
+	MetricSourceKubeAPIserverMetrics
+	MetricSourceKubeControllerManager
+	MetricSourceKubeDNS
+	MetricSourceKubeMetricsServer
+	MetricSourceKubeProxy
+	MetricSourceKubeScheduler
+	MetricSourceKubelet
+	MetricSourceKubernetesState
+	MetricSourceKyototycoon
+	MetricSourceLighttpd
+	MetricSourceLinkerd
+	MetricSourceLinuxProcExtras
+	MetricSourceMapr
+	MetricSourceMapreduce
+	MetricSourceMarathon
+	MetricSourceMarklogic
+	MetricSourceMcache
+	MetricSourceMesosMaster
+	MetricSourceMesosSlave
+	MetricSourceMongo
+	MetricSourceMysql
+	MetricSourceNagios
+	MetricSourceNfsstat
+	MetricSourceNginx
+	MetricSourceNginxIngressController
+	MetricSourceOpenldap
+	MetricSourceOpenmetrics
+	MetricSourceOpenstack
+	MetricSourceOpenstackController
+	MetricSourceOracle
+	MetricSourcePdhCheck
+	MetricSourcePgbouncer
+	MetricSourcePhpFpm
+	MetricSourcePostfix
+	MetricSourcePostgres
+	MetricSourcePowerdnsRecursor
+	MetricSourceProcess
+	MetricSourcePrometheus
+	MetricSourceProxysql
+	MetricSourcePulsar
+	MetricSourceRabbitmq
+	MetricSourceRedisdb
+	MetricSourceRethinkdb
+	MetricSourceRiak
+	MetricSourceRiakcs
+	MetricSourceSapHana
+	MetricSourceScylla
+	MetricSourceSilk
+	MetricSourceSinglestore
+	MetricSourceSnowflake
+	MetricSourceSpark
+	MetricSourceSqlserver
+	MetricSourceSquid
+	MetricSourceSSHCheck
+	MetricSourceStatsd
+	MetricSourceSupervisord
+	MetricSourceSystemCore
+	MetricSourceSystemSwap
+	MetricSourceTCPCheck
+	MetricSourceTeamcity
+	MetricSourceTeradata
+	MetricSourceTLS
+	MetricSourceTokumx
+	MetricSourceTrafficServer
+	MetricSourceTwemproxy
+	MetricSourceTwistlock
+	MetricSourceVarnish
+	MetricSourceVault
+	MetricSourceVertica
+	MetricSourceVoltdb
+	MetricSourceVsphere
+	MetricSourceWin32EventLog
+	MetricSourceWindowsPerformanceCounters
+	MetricSourceWindowsService
+	MetricSourceWmiCheck
+	MetricSourceYarn
+	MetricSourceZk
```

Comment:
Related to our discussion: you're missing a couple of checks (tekton, strimzi, cloudera, ray, torchserve or temporal to name a few) I believe

Reply:
That's right! These are all checks that do not have a protobuf value in [origin.proto](https://github.com/DataDog/dd-source/blob/9ab55c73336e28013cd749401cdcdff0e37e43d2/domains/metrics/shared/libs/proto/origin/origin.proto#L493). I am in the process of including them in a separate [PR](https://github.com/DataDog/dd-source/pull/99385/files).